### ISSUE TITLE
unrecognized arguments

### ISSUE THREAD

**Issue by romildo** [SuperSandro2000/nixpkgs-review-checks#3]
When following the instructions in the [README.md](https://github.com/SuperSandro2000/nixpkgs-review-checks/blob/master/README.md) file to review a PR, I am getting an _unrecognized arguments_ error:

``` console
$ nix-shell -p nix-review cached-nix-shell gh jq

$ source /alt/cloned/nixpkgs-review-checks/bashrc 

$ nixpkgs-review pr 114567
usage: /nix/store/aa8lw31r9b9dkls0ybwrh60ahz2l37z5-nixpkgs-review-2.5.0/bin/nixpkgs-review [-h] {post-result,approve,comments,merge,pr,rev,wip} ...
/nix/store/aa8lw31r9b9dkls0ybwrh60ahz2l37z5-nixpkgs-review-2.5.0/bin/nixpkgs-review: error: unrecognized arguments: --skip-package --skip-package digikam --skip-package envoy --skip-package iosevka --skip-package librealsense --skip-package libreoffice --skip-package lumo --skip-package pcl --skip-package mrtrix --skip-package simpleitk --skip-package smesh --skip-package tts --skip-package-regex .*ceph.* --skip-package-regex freecad --skip-package-regex opencascade.* --skip-package-regex .*pytorch.* --skip-package-regex .*sage.* --skip-package-regex samba4?Full --skip-package-regex .*tensorflow.* --skip-package-regex qgis.* --skip-package-regex .*vtk.* --skip-package-regex .*wine.* --skip-package-regex python\d+Packages\.baselines --skip-package-regex python\d+Packages\.edward --skip-package-regex python\d+Packages\.mask-rcnn --skip-package-regex python\d+Packages\.pyro-ppl --skip-package-regex python\d+Packages\.scikit-tda --skip-package-regex python\d+Packages\.torchgpipe --skip-package-regex python\d+Packages\.torchvision --skip-package-regex python\d+Packages\.tflearn --skip-package-regex python\d+Packages\.umap-learn 114567
```

**Comment by SuperSandro2000** [comment]
You need to use nixpkgs-review master right now. The feature I am using should be available with the next release after 2.5.0.

**Comment by Artturin** [comment]
nixpkgs-review is at 2.6.1 so this can be closed
